apps/remix-ide/src/app/components/plugin-manager-component.js

```diff
@@ -84,112 +22,89 @@ const profile = {
 class PluginManagerComponent extends ViewPlugin {
   constructor (appManager, engine) {
     super(profile)
-    this.event = new EventEmitter()
     this.appManager = appManager
+    this.engine = engine
+    this.pluginManagerSettings = new PluginManagerSettings()
+    this.htmlElement = document.createElement('div')
+    this.htmlElement.setAttribute('id', 'pluginManager')
     this.views = {
       root: null,
       items: {}
     }
-    this.localPlugin = new LocalPlugin()
     this.filter = ''
-    this.appManager.event.on('activate', () => { this.reRender() })
-    this.appManager.event.on('deactivate', () => { this.reRender() })
-    this.engine = engine
-    this.engine.event.on('onRegistration', () => { this.reRender() })
+    this.pluginNames = this.appManager.actives
+    this.activePlugins = []
+    this.inactivePlugins = []
+    this.activeProfiles = this.appManager.actives
```

Comment:
Duplicate declaration on line 35. Any reason for that?

Reply:
Removed.